### PR TITLE
[EDQ-437] Modified the 'Data Quality Tabs' Script to Include New Metrics

### PR DESCRIPTION
**Pull request by noahgengel**
* Implemented the three new NIH metrics into the dictionaries and other sources
* Also includes improvements to the visualizations scripts (e.g. pre-determined scales, matplotlib inline)
* Also modest improvements to the e-mail generator (more informative messages, typo fixes, etc.)

### CODE REVIEW

File: data_steward/analytics/tools/visualizations/completeness/ingredient_usage_visualizations.py [hunk 1]
```diff
@@ -101,7 +104,7 @@
 print(len(site_name_list))
 
 # +
-name_of_interest = 'aggregate_info'
+name_of_interest = 'nyc_cu'
```
Comment (nishanthpp93) [inline data_steward/analytics/tools/visualizations/completeness/ingredient_usage_visualizations.py:108]: Is this supposed to be hardcoded to only work for `nyc_cu`?

File: data_steward/analytics/tools/dq_issue_tracker/class_definitions.py [hunk 2]
```diff
@@ -257,6 +291,21 @@ def find_failing_metrics(self):
                     thresholds['route_success_min']:
                 failing_metrics.append(ingredient_integration_obj)
 
+        for date_disparity_obj in self.date_datetime_disparity:
+            if date_disparity_obj.value > \
+                    thresholds['date_datetime_disparity_max']:
+                failing_metrics.append(date_disparity_obj)
```
Comment (nishanthpp93) [inline data_steward/analytics/tools/dq_issue_tracker/class_definitions.py:297]: Not sure if it's necessary to split this line.

File: data_steward/analytics/tools/visualizations/completeness/ingredient_usage_visualizations.py (hunk 1 above)
Comment (noahgengel) [reply data_steward/analytics/tools/visualizations/completeness/ingredient_usage_visualizations.py:108]: no it is something that we change to generate the graphs for each of the sites. looks like I happened to push when I just generated the graphs for nyc_cu. I can change this in the next PR but it does not affect functionality.

File: data_steward/analytics/tools/dq_issue_tracker/class_definitions.py (hunk 2 above)
Comment (noahgengel) [reply data_steward/analytics/tools/dq_issue_tracker/class_definitions.py:297]: can change this in EDQ-440 as well

**Pull request merged**